### DISCUSSION TITLE
希望在首页能把特定筛选规则保存成自定义标签栏（跟“个性推荐”“正在关注”等并列）

### DISCUSSION THREAD

**Discussion by akira0245** [BewlyBewly/BewlyBewly#885] [Ideas]
希望在首页能把特定筛选规则保存成自定义标签栏（跟“个性推荐”“正在关注”等并列）
内部可以自由设定并保存过滤规则，除时长、播放量过滤外也方便加入关键词、标签、分类过滤等功能。

**Comment by hakadao** [comment]
不考慮, 另外過濾條件原理是直接前端過濾, 如果條件太苛刻的話會頻繁調用b站接口, 不建議過度使用 (隔壁 bilibili app recommend 也是會頻繁調用接口)
